### ISSUE TITLE
The 1.18 version's chairs spawn pigs.

### ISSUE THREAD

**Issue by FruitSanwich** [skyjay1/Tanuki-Decor#6]
_I'm on my phone right now so I can't add any logs or anything._
When I try to sit on the furniture in this mod it spawns a pig. I'm currently using this mod in a modpack, but I plan to test it again separately to see if the pig spawning is a result of some conflict with another mod in the pack.
In the meantime I figured I'd say something, just in case someone else finds a fix before I do. If it turns out it's just a conflict I'll come back and edit this appropriately to reflect the issue and solution (if I find it).

**Comment by skyjay1** [comment]
Yes, in order for the player to sit, there must be an entity. The pig should be silent and invisible, is that not the case?

**Comment by FruitSanwich** [comment]
No, it quite literally just spawns in normal pigs. The player doesn't actually sit down; instead the player is teleported near the seat, a dismount prompt pops up, and a pig spawns. It's odd but it seemingly only happens in this version. I tried newer versions and it doesn't happen.
